osbs/tekton.py

```diff
@@ -507,9 +507,9 @@ def get_error_message(self):
 
         if plugin_errors:
             for plugin, error in plugin_errors.items():
-                err_message += f"Error in plugin {plugin}: {error}\n"
+                err_message += f"Error in plugin {plugin}: {error};\n"
 
-        err_message += "\npipeline run errors:\n"
+        err_message += "\npipeline run failed;\n"
```

Comment:
Do we want this here in the middle/start of the error message? I think most of the time, only `Error in plugin ...` and `Error in <task> ...` will be relevant.

Maybe, if we go through plugin_errors and all taskRuns and still have no error message, we could then add this at the end?  Something like `pipeline run failed for unknown reason`?

Reply:
ok